pokedex/main.py

```diff
@@ -209,6 +209,19 @@ def command_dump(parser, args):
 
     if args.langs is not None:
```

Comment:
Something like this:

```py
if args.langs == 'none':
    langs = []
elif args.langs is None or args.langs == 'all':
    langs = None
else:
    langs = [l.strip() for l in args.langs.split(',')]
    # etc...
```

Reply:
If I do something like this then in load.py I would have to change
```
elif langs == ['none']:
    return False
```
to
```
elif len(langs) == 0:
    return False
```
So I would think that its pretty much the same thing